src/library/banshee/bansheeplaylistmodel.cpp

```diff
@@ -75,71 +77,37 @@ void BansheePlaylistModel::setTableModel(int playlistId) {
+                " TEXT, " CLM_GROUPING " TEXT, " CLM_TRACKNUMBER
+                " INTEGER, " CLM_DATEADDED " INTEGER, " CLM_BPM
+                " INTEGER, " CLM_BITRATE " INTEGER, " CLM_COMMENT
+                " TEXT, " CLM_PLAYCOUNT " INTEGER, " CLM_COMPOSER
+                " TEXT, " CLM_PREVIEW " TEXT)");
         if (!query.exec(strQuery.arg(m_tempTableName))) {
             LOG_FAILED_QUERY(query);
         }
 
-        QString strQuery2("INSERT INTO %1"
-                " (" CLM_TRACK_ID ", "
-                     CLM_VIEW_ORDER ", "
-                     CLM_ARTIST ", "
-                     CLM_TITLE ", "
-                     CLM_DURATION ", "
-                     CLM_URI ", "
-                     CLM_ALBUM ", "
-                     CLM_ALBUM_ARTIST ", "
-                     CLM_YEAR ", "
-                     CLM_RATING ", "
-                     CLM_GENRE ", "
-                     CLM_GROUPING ", "
-                     CLM_TRACKNUMBER ", "
-                     CLM_DATEADDED ", "
-                     CLM_BPM ", "
-                     CLM_BITRATE ", "
-                     CLM_COMMENT ", "
-                     CLM_PLAYCOUNT ", "
-                     CLM_COMPOSER ") "
-                     "VALUES (:"
-                     CLM_TRACK_ID ", :"
-                     CLM_VIEW_ORDER ", :"
-                     CLM_ARTIST ", :"
-                     CLM_TITLE ", :"
-                     CLM_DURATION ", :"
-                     CLM_URI ", :"
-                     CLM_ALBUM ", :"
-                     CLM_ALBUM_ARTIST ", :"
-                     CLM_YEAR ", :"
-                     CLM_RATING ", :"
-                     CLM_GENRE ", :"
-                     CLM_GROUPING ", :"
-                     CLM_TRACKNUMBER ", :"
-                     CLM_DATEADDED ", :"
-                     CLM_BPM ", :"
-                     CLM_BITRATE ", :"
-                     CLM_COMMENT ", :"
-                     CLM_PLAYCOUNT ", :"
-                     CLM_COMPOSER ") ");
+        QString strQuery2(
+                "INSERT INTO %1"
+                " (" CLM_TRACK_ID ", " CLM_VIEW_ORDER ", " CLM_ARTIST
+                ", " CLM_TITLE ", " CLM_DURATION ", " CLM_URI ", " CLM_ALBUM
+                ", " CLM_ALBUM_ARTIST ", " CLM_YEAR ", " CLM_RATING
+                ", " CLM_GENRE ", " CLM_GROUPING ", " CLM_TRACKNUMBER
+                ", " CLM_DATEADDED ", " CLM_BPM ", " CLM_BITRATE
+                ", " CLM_COMMENT ", " CLM_PLAYCOUNT ", " CLM_COMPOSER
+                ") "
+                "VALUES (:" CLM_TRACK_ID ", :" CLM_VIEW_ORDER ", :" CLM_ARTIST
+                ", :" CLM_TITLE ", :" CLM_DURATION ", :" CLM_URI ", :" CLM_ALBUM
+                ", :" CLM_ALBUM_ARTIST ", :" CLM_YEAR ", :" CLM_RATING
+                ", :" CLM_GENRE ", :" CLM_GROUPING ", :" CLM_TRACKNUMBER
+                ", :" CLM_DATEADDED ", :" CLM_BPM ", :" CLM_BITRATE
+                ", :" CLM_COMMENT ", :" CLM_PLAYCOUNT ", :" CLM_COMPOSER ") ");
```

Comment:
the same here, please revert.

Reply:
The same issue as in the other PR, the pre-commit hook kicked in.